### PR TITLE
ddl: Pass race

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @coocood @ngaut

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -153,9 +153,7 @@ func asyncNotify(ch chan struct{}) {
 	}
 }
 
-// Background job is serial processing, so we can extend the owner timeout to make sure
-// a batch of rows will be processed before timeout.
-var minBgOwnerTimeout = 20 * time.Second
+var maxBgOwnerTimeout = int64(10 * time.Minute)
```
Comment (ngaut) [inline ddl/ddl_worker.go]: Define as constant

**Comment by zimulala** [comment]
PTAL @ngaut @coocood @tiancaiamao

**Comment by ngaut** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**